client/network/src/service.rs

```diff
@@ -844,14 +823,10 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkService<B, H> {
 	// TODO: remove this method after https://github.com/paritytech/substrate/issues/4587
```

Comment:
https://github.com/paritytech/substrate/issues/4587 seems to be closed, can the TODO be resolved now?

Reply:
Err. It got closed, but not in a satisfying way.
I'll change this TODO to redirect to https://github.com/paritytech/substrate/issues/6827 instead.